src/cmake/android_openssl.cmake

```diff
@@ -54,6 +56,26 @@ add_custom_command(
         COMMAND ${CMAKE_COMMAND} -E copy ${_OPENSSL_CRYPTO_MODULE}/libs/android.${ANDROID_ABI}/libcrypto.so ${_OPENSSL_LIBS}/libcrypto.so)
 
 
-get_property(crypto_module GLOBAL PROPERTY OPENSSL_CRYPTO_MODULE)
-get_property(ssl_module GLOBAL PROPERTY OPENSSL_SSL_MODULE)
-get_property(openssl_libs GLOBAL PROPERTY OPENSSL_LIBS)
+# In case of newer versions of QT, for QtSLL we need to 
+# have 3.x.x bundled alongside.
+# Some rust crates seem to need 1.x? so ugh. both i guess. 
```

Comment:
Did you check if this is a config thing or something we can update on the Rust side at all?

Reply:
theorecially the openssl bindings should support that i think